doc/lua_api.txt

```diff
@@ -1808,7 +1808,7 @@ Example:
 Item Metadata
 -------------
 
-Item stacks can store metadata too. See `ItemStackMetaRef`.
+Item stacks can store metadata too. See [`ItemStackMetaRef`].
```

Comment:
No ``

Reply:
This is referring to the section in the class reference which does have the backticks, so it must be the same to be linked.